cli/src/templates/packages/expo-router/stack/app/_layout.tsx.ejs

```diff
@@ -1,16 +1,34 @@
 <% if (props.stylingPackage?.name === "nativewind") { %>
 	import { Stack } from "expo-router";
 <% } else if (props.stylingPackage?.name === "tamagui") { %>
-	import { TamaguiProvider } from "tamagui";
-	import { Stack } from "expo-router";
+  import { Stack } from "expo-router";
```

Comment:
it looks like `import { Stack } from "expo-router";` is imported regardless of the choice of styling, we should move it out of these if statements.